src/toolkits/activity_classification/ac_data_iterator.hpp

```diff
@@ -108,6 +109,15 @@ class data_iterator {
      */
     neural_net::shared_float_array weights;
 
+
+    /**
+     * An array with shape: (requested_batch_size, 
+     * 1, prediction_window * predictions_in_chunk, 1)
+     *
+     * Each row is the sequence of raw class labels (indices) corresponding every feature.
+     */
+    neural_net::shared_float_array ground_truth;
```

Comment:
Let's call this something like `labels_per_row`, so it's more obvious that this is really the same thing as `labels` but more granular

Reply:
Yeah I think that would be more intuitive!